packages/tests/server/interop/react/__testHelpers.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Any particular reason for this one? I tried switching the `{}` to `object` in it and didn't see any build failures.